ftest/nightwatchjs/itests.xml

```diff
@@ -0,0 +1,42 @@
+<project name="nightwatch-tests" xmlns:artifact="urn:nuxeo-artifact" xmlns:if="ant:if" xmlns:unless="ant:unless">
+  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build"/>
+  <taskdef resource="org/nuxeo/build/artifact/antlib.xml" uri="urn:nuxeo-artifact"/>
+  <taskdef resource="net/sf/antcontrib/antlib.xml"/>
+
+  <property name="project.dir" value="${maven.basedir}"/>
+  <property name="out.dir" value="${maven.project.build.directory}"/>
+  <unzip dest="${out.dir}/" overwrite="false">
+    <artifact:resolveFile key="org.nuxeo:nuxeo-ftest::zip"/>
+  </unzip>
+  <import file="${out.dir}/nuxeo-ftest.xml"/>
+
+  <!-- Instruction for installing the MP to be tested and selenium to be run -->
+  <property name="mp.install" value="file:${out.dir}/thesource-marketplace-${maven.project.version}.zip"/>
+  <!-- Override default prepare-environment target to make it download your MP -->
+  <target name="prepare-environment" depends="_init,prepare-db,prepare-tomcat,prepare-jboss">
+    <condition property="cmd.npm" value="npm.cmd" else="npm">
+      <os family="windows"/>
+    </condition>
+    <exec executable="${cmd.npm}" failonerror="true">
+      <arg value="install"/>
+    </exec>
+    <exec executable="${cmd.npm}" spawn="true">
+      <arg value="start"/>
+    </exec>
+    <copy todir="${out.dir}">
+      <artifact:file key="org.nuxeo.ecm.distribution:nuxeo-marketplace-jsf-ui::zip" />
```

Comment:
The `nuxeo-marketplace-jsf-ui` is retrieved and copied here, but it's never installed. Default distribution in functional tests is now a Nuxeo Server.